app/sidekiq/evss/disability_compensation_form/submit_form0781.rb

```diff
@@ -41,7 +41,7 @@ class SubmitForm0781 < Job
       # Sidekiq has built in exponential back-off functionality for retries
       # A max retry attempt of 10 will result in a run time of ~8 hours
       # This job is invoked from 526 background job
-      RETRY = 10
```

Comment:
This is pedantic, but should we update the comments to indicate how long the run time will be?

Reply:
We do in a million other places (for a value of 16) but I see what you are saying because we have the comment above stating 10 == ~8hrs. I feel like I would rather remove the comment since this may continue to evolve. But I will change the comment to match.

Reply:
addressed in 42b7930928